tensorflow/compiler/xla/service/gpu/tests/gpu_index_test.cc

```diff
@@ -120,6 +120,8 @@ TEST_F(GpuIndexTest,
                      /*match_optimized_ir=*/false);
 }
 
+#if TENSORFLOW_USE_ROCM
```

Comment:
Is it possible to skip it at runtime and not with ifdef?

Reply:
@cheshire We could do it through the BUILD file by adding a no_rocm tag, but that would disable all the tests in this file, which isn't what we wanted to do.